ansible/playbooks/init-nix.yml

```diff
@@ -0,0 +1,13 @@
+- name: Setup Nix
+  hosts: all
+  tasks:
+
+    - name: Download Nix installer
+      get_url:
+        url: https://nixos.org/nix/install
+        dest: /tmp/install-nix.sh
+
+    - name: Run Nix installer
+      command: /bin/bash /tmp/install-nix.sh --no-daemon
+      args:
+        creates: /nix
```

Comment:
After logging into the instance and trying to start nix-shell I get:
```bash
[ec2-user@ip-172-31-52-66 ~]$ nix-shell
error: path '/home/ec2-user/default.nix' does not exist
```
Is there a sense in setting up this file as a part of this recipe?

Reply:
Interesting.  Maybe we put a simple nix config that has some basics to get somebody started.  At the very least, putting git in there is kind of useful.